packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -339,4 +339,42 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
       })
     })
   })
+
```

Comment:
I updated the description in the PR (probably while you were reviewing). I am asking Ryan to sign off on the design changes here before we merge, we should update the test requirement too. I'm gonna put this in Draft until Ryan confirms the UI is OK (at least for now). Since retrying won't do anything and the state goes away automatically, I think the button is not really useful.

Reply:
Updated the checklist